pkg/kubectl/categories/categories.go

```diff
@@ -129,12 +140,16 @@ func (e discoveryFilteredExpander) Expand(category string) ([]schema.GroupResour
 	return available, ok
 }
 
+// UnionCategoryExpander represents for a list of CategoryExpander,
```

Comment:
UnionCategoryExpander implements CategoryExpander interface. It maps given category string to union of expansions returned by all the CategoryExpanders in the list.